types/aws-lambda/index.d.ts

```diff
@@ -501,6 +501,52 @@ export interface AuthResponseContext {
     [name: string]: any;
 }
 
+/**
+ * CodePipeline events
+ * https://docs.aws.amazon.com/codepipeline/latest/userguide/actions-invoke-lambda-function.html
+ */
+export interface S3ArtifactLocation {
+    bucketName: string;
+    objectKey: string;
+}
+export interface S3ArtifactStore {
+    type: 'S3';
+    s3Location: S3ArtifactLocation;
+}
+
+export type ArtifactLocation = S3ArtifactStore;
+
+export interface Artifact {
+    name: string;
+    revision: string | null;
+    location: ArtifactLocation;
+}
+
+export interface Credentials {
+    accessKeyId: string;
+    secretAccessKey: string;
+    sessionToken?: string;
+}
+
+export interface CodePipelineEvent {
+    "CodePipeline.job": {
+        id: string;
+        accountId: string;
+        data: {
+            actionConfiguration: {
+                configuration: {
+                    FunctionName: string;
+                    UserParameters: string;
+                }
+            };
+            inputArtifacts: Artifact[];
+            outputArtifacts: Artifact[];
+            artifactCredentials: Credentials;
+            continuationToken?: string;
```

Comment:
From the [JobData](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_JobData.html) docs, looks like you're missing `encryptionKey` (remembering the lambda event docs call out `actionTypeId` and `pipelineContext` as not included)

Reply:
I'll add `encryptionKey`.  It wasn't something I'd seen, but I wasn't using encrypted artifacts.